kim/src/kim_translation

```diff
@@ -0,0 +1,120 @@
+
```

Comment:
Cool solution on I18N! Do you know about [Internationalizing Those Bash Scripts ](https://www.linuxjournal.com/content/internationalizing-those-bash-scripts) ?  The procedure there is based on standard message catalogs. I'm not sure whether this warrants the effort, but it might be more flexible with respect to locale detection.

Reply:
Sorry, no idea about it.  My i18n scheme was written around 5 or 6 years ago when I was trying to embed kim4 into our own customized Linux distribution.

Reply:
Usually I ted to favor standard solution before own implementations. IMHO this usually helps avoid issues in the long run. But I was merely pointing out the alternative. I guess the owner should decide on how to proceed here. At the end: no whatsover perfect theoretical concept can beat a working PR.

Reply:
This implementation was also used to teach students about the most basic i18n concept: extracting messages from the program and replacing by message indexes.  Anyway I'm trying to use gettext in bash now.